woocommerce/woocommerce-ios
# Customize table view style for list selector

Update ListSelectorViewController's table view from xib to code to enable DI'ing the table view style since it's set in initialization for #2037 

## Changes

- Replaced `ListSelectorViewController.xib` with a table view manually set up in `ListSelectorViewController`
- DI'ed `tableViewStyle: UITableView.Style` to `ListSelectorViewController` with default of grouped style
- In `FilterListViewController`, set up the list selectors for the filters and filter options to have plain table view style
- In `ListSelectorViewController`, moved `command.configureCell` to after the default cell `accessoryType` setup so that it could override the `accessoryType` (e.g. to display a disclosure indicator in the filter list selector)
- In `PaginatedListSelectorViewController`, fixed the nib name in initializer (was using `ListSelectorViewController.xib` before) and set the table view style from `plain` to `grouped` in its xib

## Testing

- Go to the Products tab
- Tap the "Filter" CTA --> the filter list selector should have **plain** table view style
- Tap on a filter type row --> the filter option selector should have **plain** table view style
- Tap "Dismiss" to dismiss the filter products UI
- Tap on a simple Product
- Tap on the price row
- Tap on the tax status row --> the list selector should have **grouped** table view style
- Leave the tax status selector back to the price settings
- Tap on the tax class row --> the paginated list selector should have **grouped** table view style

## Example screenshots

mode \ screen | filter products | filter products by product type | product tax status
-- | -- | -- | --
dark | ![Simulator Screen Shot - iPhone 11 - 2020-05-05 at 09 07 08](https://user-images.githubusercontent.com/1945542/81027715-571b8b80-8eb1-11ea-82bf-70b08cd3f189.png) | ![Simulator Screen Shot - iPhone 11 - 2020-05-05 at 09 07 18](https://user-images.githubusercontent.com/1945542/81027685-423ef800-8eb1-11ea-8e62-e39f906a22e4.png) | ![Simulator Screen Shot - iPhone 11 - 2020-05-05 at 09 07 39](https://user-images.githubusercontent.com/1945542/81027688-4539e880-8eb1-11ea-8a3c-96204c98cfbd.png)
light | ![Simulator Screen Shot - iPhone 11 - 2020-05-05 at 09 07 59](https://user-images.githubusercontent.com/1945542/81027689-4703ac00-8eb1-11ea-88c4-f04d85c7f20d.png) | ![Simulator Screen Shot - iPhone 11 - 2020-05-05 at 09 08 22](https://user-images.githubusercontent.com/1945542/81027694-4834d900-8eb1-11ea-8c19-f79c266e88d5.png) | ![Simulator Screen Shot - iPhone 11 - 2020-05-05 at 09 08 30](https://user-images.githubusercontent.com/1945542/81027696-49660600-8eb1-11ea-94ff-36a821b1557b.png)


Update release notes:
